FirstTextGame.py

```diff
@@ -0,0 +1,80 @@
+# name: Noel Savoy
+# date: 4/13/2020
+# description: Text-based fantasy game
+import random
+import time
+
+def displayintro():
```

Comment:
Nice. What might be fun is to write a list of things you want to print to the screen, store it in a function and iterate through it,

Something like

``
sentences = ["They came at night, wrapped in shadow.", "You scamper from your tent.", "A masked man stands in the distance."]
for sentence in sentences:
    print(sentence)
    time.sleep(1.75)
``

Reply:
You could also write it out one character at a time, for a sort of scrolling effect